bevy_asset_loader/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bevy_asset_loader"
-version = "0.11.0"
+version = "0.11.1"
```

Comment:
No need to bump this version. There are already changes on `main` that will require a minor bump with the next release.